catroid/build.gradle

```diff
@@ -108,21 +104,10 @@ ext.copyGoogleServicesFile = { flavorName, flavorId ->
         into "src/$flavorName"
         filter { line ->
             line.replaceAll('@appId@', flavorId)
-                .replaceAll('@apiKey@', getProperty('firebaseApiKey', '@apiKey@'))
```

Comment:
I think firebaseApiKey is still used by google analytics, unless you want to remove that as well.
So, google-services.json is also still needed.

https://developers.google.com/analytics/devguides/collection/android/v4/#set_up_your_project

Reply:
Google analytics shall be deleted as well.

Reply:
then the whole google-services.json file may be deleted, right? if we dont use google analytics?